python/benchmark/test_gen_data.py

```diff
@@ -63,3 +64,39 @@ def test_make_blobs() -> None:
         cluster_stds = [0.7] * 3
         for i, (ctr, std) in enumerate(zip(centers, cluster_stds)):
             assert_almost_equal((X[y == i] - ctr).std(), std, 1, "Unexpected std")
+
+
+def test_make_low_rank_matrix() -> None:
+    input_args = [
+        "--num_rows",
+        "50",
+        "--num_cols",
+        "20",
+        "--dtype",
+        "float64",
```

Comment:
Can you parameterize test and sweep over type float64 and float32?   See how parameterization is done in the other tests.

Reply:
Also, in test blobs.